### PR TITLE
VirtualNetworkGateway Additional Testcases

### PR DESCRIPTION
**Pull request by lrxtom2**
TestAccAzureRMVirtualNetworkGateway_standard
TestAccAzureRMVirtualNetworkGateway_vpnGw2
TestAccAzureRMVirtualNetworkGateway_vpnGw3

### CODE REVIEW

File: azurerm/resource_arm_virtual_network_gateway_test.go [hunk 1]
```diff
@@ -86,6 +86,69 @@ func TestAccAzureRMVirtualNetworkGateway_activeActive(t *testing.T) {
...
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMVirtualNetworkGatewayDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMVirtualNetworkGatewayExists("azurerm_virtual_network_gateway.test"),
```
Comment (tombuildsstuff) [inline azurerm/resource_arm_virtual_network_gateway_test.go]: `"azurerm_virtual_network_gateway.test"` -> `resourceName`

File: azurerm/resource_arm_virtual_network_gateway_test.go [hunk 2]
```diff
@@ -86,6 +86,69 @@ func TestAccAzureRMVirtualNetworkGateway_activeActive(t *testing.T) {
...
```
Comment (tombuildsstuff) [inline azurerm/resource_arm_virtual_network_gateway_test.go]: `"azurerm_virtual_network_gateway.test"` -> `resourceName`

File: azurerm/resource_arm_virtual_network_gateway_test.go [hunk 3]
```diff
@@ -86,6 +86,69 @@ func TestAccAzureRMVirtualNetworkGateway_activeActive(t *testing.T) {
...
```
Comment (tombuildsstuff) [inline azurerm/resource_arm_virtual_network_gateway_test.go]: `"azurerm_virtual_network_gateway.test"` -> `resourceName`

**Comment by lrxtom2** [comment]
Thanks for @tombuildsstuff 
I will pay attention to that next time.

**Comment by tombuildsstuff** [comment]
Tests pass, ignoring an existing test failure (which is fixed in #1431):

screenshot 2018-06-22 at 16 10 18

**Pull request merged**